app/models/merchant_user.rb

```diff
@@ -1,4 +1,5 @@
-class MerchantUser < ApplicationRecord
-  belongs_to :merchant
-  belongs_to :user 
-end
+# TO BE DELETED
+# class MerchantUser < ApplicationRecord
+#   belongs_to :merchant
+#   belongs_to :user
+# end
```

Comment:
I think moving forward if we agree as a group the file needs to be deleted, we can all just delete that file in our individual code before pushing up. Not that it's a big deal, but I have the feeling if we all do this, we'll end up with a bunch of files with just commented out code :)

Reply:
I think this is one of the reasons that this merge request has "big" (aka not able to be resolved on github) conflicts to resolve.